pontoon/base/models.py

```diff
@@ -276,36 +276,39 @@ def repository_for_path(self, path):
         else:
             return repo
 
-    def latest_activity(self, locale=None):
+    def latest_activity(self, locale=None, path=None):
```

Comment:
This is fine for now, but as we find a need for more options and filters on the set of translations I wonder about how appropriate it is to have these helpers on the Project and Locale models, especially now that they're diverging in the arguments they support.